### PR TITLE
HBASE-28389 HBase backup yarn queue parameter ignored

### PR DESCRIPTION
**Pull request by 2005hithlj**
https://issues.apache.org/jira/browse/HBASE-28389

### CODE REVIEW

**Comment by 2005hithlj** [comment]
@DieterDP-ng  @Apache9   Could you take a look? Thanks.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 48s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  1s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  |  Patch does not have any anti-patterns.  |
|||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m  5s |  |  branch-2 passed  |
| +1 :green_heart: |  compile  |   0m 30s |  |  branch-2 passed  |
| +1 :green_heart: |  checkstyle  |   0m 12s |  |  branch-2 passed  |
| +1 :green_heart: |  spotbugs  |   0m 34s |  |  branch-2 passed  |
| +1 :green_heart: |  spotless  |   0m 46s |  |  branch has no errors when running spotless:check.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 58s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 27s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 27s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   0m 12s |  |  the patch passed  |
| +1 :green_heart: |  spotbugs  |   0m 40s |  |  the patch passed  |
| +1 :green_heart: |  hadoopcheck  |  15m 39s |  |  Patch does not cause any errors with Hadoop 2.10.2 or 3.3.6 3.4.0.  |
| +1 :green_heart: |  spotless  |   0m 44s |  |  patch has no errors when running spotless:check.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 11s |  |  The patch does not generate ASF License warnings.  |
|  |   |  28m 30s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.46 ServerAPI=1.46 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6130 |
| Optional Tests | dupname asflicense javac spotbugs checkstyle codespell detsecrets compile hadoopcheck hbaseanti spotless |
| uname | Linux be2d5f8823d2 5.4.0-186-generic #206-Ubuntu SMP Fri Apr 26 12:31:10 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / a5cf8972d7c1f6139595e72c7aefbf32e06eaef7 |
| Default Java | Eclipse Adoptium-11.0.23+9 |
| Max. process+thread count | 83 (vs. ulimit of 30000) |
| modules | C: hbase-backup U: hbase-backup |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/console |
| versions | git=2.34.1 maven=3.9.8 spotbugs=4.7.3 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 42s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  6s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 38s |  |  branch-2 passed  |
| +1 :green_heart: |  compile  |   0m 19s |  |  branch-2 passed  |
| +1 :green_heart: |  javadoc  |   0m 15s |  |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   4m 59s |  |  branch has no errors when building our shaded downstream artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 30s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 18s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 18s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 14s |  |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m  1s |  |  patch has no errors when building our shaded downstream artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  11m 55s |  |  hbase-backup in the patch passed.  |
|  |   |  30m  2s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.46 ServerAPI=1.46 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/artifact/yetus-jdk8-hadoop2-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6130 |
| Optional Tests | javac javadoc unit compile shadedjars |
| uname | Linux d6bfaca083af 5.4.0-182-generic #202-Ubuntu SMP Fri Apr 26 12:29:36 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / a5cf8972d7c1f6139595e72c7aefbf32e06eaef7 |
| Default Java | Temurin-1.8.0_412-b08 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/testReport/ |
| Max. process+thread count | 3410 (vs. ulimit of 30000) |
| modules | C: hbase-backup U: hbase-backup |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 20s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  5s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m  8s |  |  branch-2 passed  |
| +1 :green_heart: |  compile  |   0m 22s |  |  branch-2 passed  |
| +1 :green_heart: |  javadoc  |   0m 17s |  |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   5m 30s |  |  branch has no errors when building our shaded downstream artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 58s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 21s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 21s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 15s |  |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 36s |  |  patch has no errors when building our shaded downstream artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  10m 40s |  |  hbase-backup in the patch passed.  |
|  |   |  30m 35s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.46 ServerAPI=1.46 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/artifact/yetus-jdk17-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6130 |
| Optional Tests | javac javadoc unit compile shadedjars |
| uname | Linux e4c3fd1a7bb8 5.4.0-182-generic #202-Ubuntu SMP Fri Apr 26 12:29:36 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / a5cf8972d7c1f6139595e72c7aefbf32e06eaef7 |
| Default Java | Eclipse Adoptium-17.0.11+9 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/testReport/ |
| Max. process+thread count | 3182 (vs. ulimit of 30000) |
| modules | C: hbase-backup U: hbase-backup |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 43s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  5s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 10s |  |  branch-2 passed  |
| +1 :green_heart: |  compile  |   0m 18s |  |  branch-2 passed  |
| +1 :green_heart: |  javadoc  |   0m 14s |  |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   5m 56s |  |  branch has no errors when building our shaded downstream artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 57s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 19s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 19s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 14s |  |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 58s |  |  patch has no errors when building our shaded downstream artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  11m 27s |  |  hbase-backup in the patch passed.  |
|  |   |  32m 26s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6130 |
| Optional Tests | javac javadoc unit compile shadedjars |
| uname | Linux 7f39f84c6acc 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / a5cf8972d7c1f6139595e72c7aefbf32e06eaef7 |
| Default Java | Eclipse Adoptium-11.0.23+9 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/testReport/ |
| Max. process+thread count | 3710 (vs. ulimit of 30000) |
| modules | C: hbase-backup U: hbase-backup |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/1/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupCommands.java [hunk 1]
```diff
@@ -335,8 +335,8 @@ public void execute() throws IOException {
 
       if (cmdline.hasOption(OPTION_YARN_QUEUE_NAME)) {
```
Comment (DieterDP-ng) [inline hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupCommands.java:336]: I suggest moving this block to before the `super.execute();` that creates the connection. That way, you can directly update the conf using `getConf()`. Won't have any change in outcome, but feels a bit simpler.

**Review by DieterDP-ng — APPROVED** [review]
Looks good, thanks for solving this.

File: hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupCommands.java (hunk 1 above)
Comment (2005hithlj) [reply hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupCommands.java]: @DieterDP-ng  Thanks for your review. I have revised it. could you take a look? Thanks.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 42s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  |  Patch does not have any anti-patterns.  |
|||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m  9s |  |  branch-2 passed  |
| +1 :green_heart: |  compile  |   0m 30s |  |  branch-2 passed  |
| +1 :green_heart: |  checkstyle  |   0m 13s |  |  branch-2 passed  |
| +1 :green_heart: |  spotbugs  |   0m 34s |  |  branch-2 passed  |
| +1 :green_heart: |  spotless  |   0m 47s |  |  branch has no errors when running spotless:check.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 57s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 27s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 27s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   0m 11s |  |  the patch passed  |
| +1 :green_heart: |  spotbugs  |   0m 39s |  |  the patch passed  |
| +1 :green_heart: |  hadoopcheck  |  15m 37s |  |  Patch does not cause any errors with Hadoop 2.10.2 or 3.3.6 3.4.0.  |
| +1 :green_heart: |  spotless  |   0m 45s |  |  patch has no errors when running spotless:check.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 11s |  |  The patch does not generate ASF License warnings.  |
|  |   |  28m 24s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.46 ServerAPI=1.46 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6130 |
| Optional Tests | dupname asflicense javac spotbugs checkstyle codespell detsecrets compile hadoopcheck hbaseanti spotless |
| uname | Linux 732c11e0be32 5.4.0-186-generic #206-Ubuntu SMP Fri Apr 26 12:31:10 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 3bbd7b060736f24be7148d1c92dea992fec2ac59 |
| Default Java | Eclipse Adoptium-11.0.23+9 |
| Max. process+thread count | 78 (vs. ulimit of 30000) |
| modules | C: hbase-backup U: hbase-backup |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/console |
| versions | git=2.34.1 maven=3.9.8 spotbugs=4.7.3 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 46s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  5s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 25s |  |  branch-2 passed  |
| +1 :green_heart: |  compile  |   0m 18s |  |  branch-2 passed  |
| +1 :green_heart: |  javadoc  |   0m 14s |  |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   5m  1s |  |  branch has no errors when building our shaded downstream artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 26s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 19s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 19s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 14s |  |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 57s |  |  patch has no errors when building our shaded downstream artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  11m 44s |  |  hbase-backup in the patch passed.  |
|  |   |  29m 36s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.46 ServerAPI=1.46 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/artifact/yetus-jdk8-hadoop2-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6130 |
| Optional Tests | javac javadoc unit compile shadedjars |
| uname | Linux 4e08f065b2b5 5.4.0-182-generic #202-Ubuntu SMP Fri Apr 26 12:29:36 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 3bbd7b060736f24be7148d1c92dea992fec2ac59 |
| Default Java | Temurin-1.8.0_412-b08 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/testReport/ |
| Max. process+thread count | 2975 (vs. ulimit of 30000) |
| modules | C: hbase-backup U: hbase-backup |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 42s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  5s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 54s |  |  branch-2 passed  |
| +1 :green_heart: |  compile  |   0m 18s |  |  branch-2 passed  |
| +1 :green_heart: |  javadoc  |   0m 14s |  |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   5m 53s |  |  branch has no errors when building our shaded downstream artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   2m 55s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 18s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 18s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 13s |  |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 56s |  |  patch has no errors when building our shaded downstream artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  11m 22s |  |  hbase-backup in the patch passed.  |
|  |   |  31m 58s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6130 |
| Optional Tests | javac javadoc unit compile shadedjars |
| uname | Linux 983f332cb4a7 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 3bbd7b060736f24be7148d1c92dea992fec2ac59 |
| Default Java | Eclipse Adoptium-11.0.23+9 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/testReport/ |
| Max. process+thread count | 3821 (vs. ulimit of 30000) |
| modules | C: hbase-backup U: hbase-backup |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 20s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  5s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 22s |  |  branch-2 passed  |
| +1 :green_heart: |  compile  |   0m 22s |  |  branch-2 passed  |
| +1 :green_heart: |  javadoc  |   0m 16s |  |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   7m  3s |  |  branch has no errors when building our shaded downstream artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   4m 20s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 30s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 30s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 21s |  |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m 14s |  |  patch has no errors when building our shaded downstream artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  13m 43s |  |  hbase-backup in the patch passed.  |
|  |   |  38m 42s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.46 ServerAPI=1.46 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/artifact/yetus-jdk17-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6130 |
| Optional Tests | javac javadoc unit compile shadedjars |
| uname | Linux 24d0a658c229 5.4.0-182-generic #202-Ubuntu SMP Fri Apr 26 12:29:36 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 3bbd7b060736f24be7148d1c92dea992fec2ac59 |
| Default Java | Eclipse Adoptium-17.0.11+9 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/testReport/ |
| Max. process+thread count | 3185 (vs. ulimit of 30000) |
| modules | C: hbase-backup U: hbase-backup |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6130/2/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupCommands.java (hunk 1 above)
Comment (DieterDP-ng) [reply hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupCommands.java]: Looks good, no further remarks from me.

**Review by Apache9 — APPROVED** [review]

**Pull request merged**